server/manager/src/main/java/org/apache/accumulo/manager/tableOps/bulkVer2/LoadFiles.java

```diff
@@ -195,17 +197,41 @@ private void sendQueued(int threshhold) {
           try {
             client = ThriftUtil.getClient(ThriftClientTypes.TABLET_SERVER, server,
                 manager.getContext(), timeInMillis);
-            client.loadFiles(TraceUtil.traceInfo(), manager.getContext().rpcCreds(), tid,
-                bulkDir.toString(), tabletFiles, setTime);
+            // Send a message per tablet. On the tablet server side for each tablet it must write to
+            // the metadata tablet which requires waiting on the walog. Sending a message per tablet
+            // allows these per tablet metadata table writes to run in parallel. This avoids
+            // serially waiting on the metadata table write for each tablet.
+            for (var entry : tabletFiles.entrySet()) {
```

Comment:
This is the code that I changed to do an RCP per tablet instead of per tablet server. I was hoping these would execute in parallel on the tablet server, but that does not seem to be happening.